end_to_end_tests/test/test_rest_api_with_authorizers.py

```diff
@@ -0,0 +1,199 @@
+import boto3
+import os
+from .helpers.base_test import BaseTest
+from .helpers.resources import Resources
+from .helpers.resource_types import ResourceTypes
+import urllib3
```

Comment:
Any reason we choice urllib3 over requests?

Reply:
No specific reason. I will update the code to use `requests` module